components/03-modules/newsletter/_newsletter.scss

```diff
@@ -0,0 +1,90 @@
+$newsletter__font-size                : $font-size-base !default;
+$newsletter__font-size--agreements    : $font-size-extra-small !default;
+$newsletter__line-height              : 1.2 !default;
+$newsletter__input-background         : #717171 !default;
+$newsletter__input-placholder-opacity : 0.7 !default;
+$newsletter__heading-color            : #ffffff;
+$newsletter__controls-height          : 32px;
+
+.newsletter {
+    &__heading {
+        line-height: $newsletter__line-height;
+        margin-right: 20px;
+        color: $newsletter__heading-color;
+        font-family: $font-family-sans-serif;
+        height: 18px;
+        font-size: $newsletter__font-size;
+        font-weight: bold;
+        text-align: left;
+        margin-bottom: 28px;
+    }
+    &__controls {
+        position: relative;
+        height: $newsletter__controls-height;
+    }
+    &__button {
+        position: absolute;
+        top: 0;
+        right: 0;
+        height: $newsletter__controls-height;
+        border-radius: 100px;
+        padding-left: 21px;
+        padding-right: 21px;
+        &:hover,
+        &:focus {
+            cursor: pointer;
+            outline: none;
+        }
+    }
+    &__input {
+        margin: 0;
+    }
+    &__label {
+        height: 0;
+    }
+    &__field {
+        background: $newsletter__input-background;
+        padding: 7px 16px;
```

Comment:
Can we use base padding `8x` instead of `7px`?

Reply:
in project is 7px. Should I use 8px anyway ?

Reply:
changed to default values

Reply:
👍